tests/method/tmapper.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-  errormsg: "invalid declaration order; cannot attach 'step' to method defined here: tmapper.nim(22,7)"
+  errormsg: "invalid declaration order; cannot attach 'step' to method defined here: tests/method/tmapper.nim(22,7)"
```

Comment:
Actually on Windows the path is just tmapper.nim(22,7). We need to get rid of this discrepancy.